premailer/tests/test_premailer.py

```diff
@@ -1828,6 +1829,19 @@ def test_load_external_url(self, mocked_requests):
         mocked_requests.get.assert_called_once_with(faux_uri)
         eq_(faux_response, r)
 
+    @mock.path('premailer.premailer.requests')
```

Comment:
Ha! I got too lazy to run the tests. tox was giving me trouble. I'll fix them and run the tests tomorrow :)